app/views/guitars/guitar_gallery.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Guitars#guitarGallery</h1>
+<p>Find me in app/views/guitars/guitarGallery.html.erb</p>
```

Comment:
this autogenerated line can be removed, same for the welcome_page

Reply:
I'll remove them @MariaAga 👍🏼